specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/orchestratorInstances.json

```diff
@@ -0,0 +1,540 @@
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "An identifier that represents the resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ResourceUpdateParameters": {
+      "description": "Parameters for updating a resource.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      }
+    },
+    "OrchestratorInstances": {
+      "description": "An array of OrchestratorInstance resources.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OrchestratorInstance"
+          },
+          "description": "An array of OrchestratorInstance resources."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "OrchestratorInstance": {
+      "description": "Represents an instance of a orchestrator.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "readOnly": true,
+          "$ref": "#/definitions/OrchestratorResourceProperties",
+          "description": "Properties of the provision operation request."
+        }
+      }
+    },
+    "OrchestratorResourceProperties": {
+      "description": "Properties of orchestrator",
+      "type": "object",
+      "properties": {
+        "resourceGuid": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource guid."
+        },
+        "state": {
+          "type": "string",
+          "enum": [
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "name": "orchestratorInstance state",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "The current state of orchestratorInstance resource."
+        },
+        "orchestratorInstanceDetails": {
+          "description": "orchestrator details",
+          "$ref": "#/definitions/OrchestratorInstanceDetails"
+        },
+        "controllerDetails": {
```

Comment:
same question as subnets, can this be a child of a controller if it is associated with one?